RAReorderableLayout/RAReorderableLayout.swift

```diff
@@ -582,9 +580,15 @@ private class RACellFakeView: UIView {
     
     private func getCellImage() -> UIImage {
         UIGraphicsBeginImageContextWithOptions(cell!.bounds.size, false, UIScreen.mainScreen().scale * 2)
+        defer { UIGraphicsEndImageContext() }
+
         cell!.drawViewHierarchyInRect(cell!.bounds, afterScreenUpdates: true)
-        let image: UIImage = UIGraphicsGetImageFromCurrentImageContext()
-        UIGraphicsEndImageContext()
-        return image
+        return UIGraphicsGetImageFromCurrentImageContext()
     }
 }
+
+// Convenience method
+func ~= (obj:NSObjectProtocol?, r:UIGestureRecognizer) -> Bool
+{
+    return r.isEqual(obj)
+}
```

Comment:
Sorry for late reply. The code you mentioned is only used in order to write `switch` statement (for gesture recognizer delegate methods) without `!`.

```swift
switch gestureRecognizer {
   case longPress: ...  /* if there isn't the convenience method, `!` is needed because these are optional vars. */
   case panGesture: ... // same
   :
```   
If it is removed, I think we should revert the `switch` code to `if` statements. If you have more better idea, I would like to know.